pylib/anki/scheduler/base.py

```diff
@@ -15,6 +15,8 @@
 UnburyDeck = scheduler_pb2.UnburyDeckRequest
 BuryOrSuspend = scheduler_pb2.BuryOrSuspendCardsRequest
 CustomStudyRequest = scheduler_pb2.CustomStudyRequest
+ScheduleCardsAsNew = scheduler_pb2.ScheduleCardsAsNewRequest
+ScheduleCardsAsNewDefaults = scheduler_pb2.ScheduleCardsAsNewDefaultsResponse
```

Comment:
Renaming these makes it a bit more friendly for the frontend code, and less likely that they'll accidentally import from our semi-private _pb2 files.